e2e/tests/lineChart/lineChart-sidebar.e2e-spec.ts

```diff
@@ -1,67 +1,65 @@
-    await sidebar.show.clickOnCountryFromList('Argentina', true);
-    expect(await lineChart.getSelectedCountriesNames()).toMatch('Argentina');
-
-    expect(await lineChart.countriesLines.count()).toEqual(DEFAULT_COUNTRIES_NUMBER + 1);
-  });
+    const DEFAULT_COUNTRIES_NUMBER = 4;
+    beforeEach(async () => {
+        await lineChart.openChart();
+    });
+    it('Add country from country list in sidebar', async () => {
+        await sidebar.show.clickOnCountryFromList('Argentina', true);
+        expect(await lineChart.getSelectedCountriesNames()).toMatch('Argentina');
 
-  it('Add country from search in sidebar', async () => {
-    await sidebar.show.searchAndSelectCountry('Argentina');
-    expect(await lineChart.getSelectedCountriesNames()).toMatch('Argentina');
+        expect(await lineChart.countriesLines.count()).toEqual(DEFAULT_COUNTRIES_NUMBER + 1);
+    });
 
-    expect(await lineChart.countriesLines.count()).toEqual(DEFAULT_COUNTRIES_NUMBER + 1);
-  });
+    it('Add country from search in sidebar', async () => {
+        await sidebar.show.searchAndSelectCountry('Argentina');
+        expect(await lineChart.getSelectedCountriesNames()).toMatch('Argentina');
 
-  it('Reset button drop settings to default', async () => {
-    await sidebar.show.clickOnCountryFromList('Argentina', true);
-    await sidebar.show.clickResetButton();
+        expect(await lineChart.countriesLines.count()).toEqual(DEFAULT_COUNTRIES_NUMBER + 1);
+    });
 
-    expect(await lineChart.countriesLines.count()).toEqual(DEFAULT_COUNTRIES_NUMBER, 'number of selected countries');
-  });
+    it('Reset button drop settings to default', async () => {
+        await sidebar.show.clickOnCountryFromList('Argentina', true);
+        await sidebar.show.clickResetButton();
 
-  xit('"Find" button in sidebar show only selected countries', async () => {
-    const chartCountries = lineChart.selectedCountries;
-    await sidebar.findSelect.clickOnFindButton();
-    const modalCountries = sidebar.findSelect.countriesInFindModal;
+        expect(await lineChart.countriesLines.count()).toEqual(DEFAULT_COUNTRIES_NUMBER, 'number of selected countries');
+    });
 
-    expect(await chartCountries.count()).toEqual(await modalCountries.count());
+    it('"Find" button in sidebar show only selected countries', async () => {
+        const chartCountries = lineChart.selectedCountries;
+        const modalCountries = sidebar.findSelect.countriesInFindModal;
 
-    /**
-     * 'United states' displayed on chart as 'United Sta...'
-     * this removes dots from name
-     * and iterate through the names to find matches
-     */
-    const chartCountriesText = await chartCountries.getText();
-    const modalCountriesText = await modalCountries.getText();
-    const filteredChartCountries = chartCountriesText
-      .toString()
-      .replace(/\./g, '')
-      .split(',');
+        expect(await chartCountries.count()).toEqual(await modalCountries.count());
```

Comment:
you check workability of "Find" button without interaction with it. How is it possible?

Reply:
Thanks for the point.
Name of the test is changed